nntrainer/src/bn_layer.cpp

```diff
@@ -92,58 +93,63 @@ int BatchNormalizationLayer::setProperty(std::vector<std::string> values) {
 
 Tensor BatchNormalizationLayer::forwarding(Tensor in, int &status) {
```

Comment:
If I am not wrong, the forward pass of batch normalization is different for training and evaluation.
So, I think you will need to update `forwarding` interface and differentiate them for train and eval;

Reply:
Yes, I need to work for that one.

Reply:
#255 has been merged. It should be helpful here.

Reply:
Thanks a lot ;)

Reply:
To keep commit size small, I will work on this issue in different PR. Thanks :)